server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -1861,8 +1861,10 @@ private CompactionStats _majorCompact(MajorCompactionReason reason) throws IOExc
       if (inputFiles.isEmpty()) {
         if (reason == MajorCompactionReason.USER) {
           // no work to do
-          lastCompactID = compactionId.getFirst();
-          updateCompactionID = true;
+//          lastCompactID = compactionId.getFirst();
```

Comment:
What was your intention in leaving these commented out?
I'm not too familiar with how the compactionId is used, but it seems like getting this right is tricky.